apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/codegen/kotlin/EnumType.kt

```diff
@@ -37,6 +38,12 @@ private val primaryConstructorSpec =
         .addParameter("rawValue", String::class)
         .build()
 
+private val sealedSubclassPrimaryConstructorSpec =
```

Comment:
The `primaryConstructorSpec` is used also when generating `enum class` and for `Unknown` case, so we need a new one that overrides the raw value

Reply:
Makes sense 👍